release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -327,6 +327,16 @@ The Node Feature Discovery (NFD) Operator detects hardware features available on
 [id="ocp-4-8-monitoring"]
 === Monitoring
 
+[id="ocp-4-8-monitoring-removed-API-alerts"]
+==== Alerts and information on APIs in use that will be removed in the next release
+
+{product-title} 4.8 introduces two new alerts that fire when an API that will be removed in the next release is in use:
```

Comment:
s/that fire/are triggered? Just thinking bout localization

Reply:
Good suggestion, though I do think that "fire" is used for alerts throughout the monitoring docs (at least the RN sections, which is where I checked for wording). I'll check w/ @lbarbeevargas to confirm!

Reply:
Got :+1: from lindsey to use "alert"